.github/workflows/ci.yml

```diff
@@ -5,14 +5,14 @@ on:
     branches:
       - main
       - main_0.1
-      - vnext-prototype
-      - vnext-tokens
+      - fluent2-tokens
```

Comment:
Since we're maintaining **main_0.2**, **main_0.3** and **main_0.4** it's probably worth adding them here as well.